src/Struct/InAppPurchaseStruct.php

```diff
@@ -17,8 +17,12 @@
 #[Package('checkout')]
 class InAppPurchaseStruct extends Struct
 {
+    public const STATUS_ACTIVE = 'active';
```

Comment:
that screams for an enum :sweat_smile:

Reply:
nice! the constants can be removed now :wink:

Reply:
hmm... they were removed. Guess I didn't add the files to the commit. But they are gone now. :)

Reply:
![](https://i.imgflip.com/9pcbcb.jpg)